_posts/history/2022-02-06-a-brief-history-of-usrse.md

```diff
@@ -0,0 +1,372 @@
+They cover all manner of topics, spanning developing professional career profiles to tools for code review.
+
+
+### Working Groups
+
+The US-RSE [governance document](https://us-rse.org/about/governance/)established a need and role for working groups, but these were informal at first.
+Between 2020 and 2021, an early set of working groups [[ref](https://us-rse.org/about/working-groups/)] including a website working group, diversity, equity, and inclusion (DEI), and education and training were more formally established with slack channels, mailing lists, and regular meetings.
+As the groups formalized, they added information on themselves to the website.
+The 2021 website refresh gave working groups a more prominent location so others could find them to join.  
+
+#### The DEI Working Group
+
+In early 2019, Sandra Gesing mentioned diversity in a brainstorming document [[ref](https://docs.google.com/document/d/1q_C86-6UCUY60A5KqY4IeE4M_kwPC5eOZYBkd_5M8nU/edit#)].
+Initial progress on the idea was made in a [September 2019 steering committee meeting](https://docs.google.com/document/d/1o6MDp3XO5LAEmkuQIJlNEdEPru4Yex4aMPpw0EikS0U/edit). During this committee meeting, there was discussion about creating a diversity statement on the US-RSE website. Dan Katz had suggested this topic, writing to the steering committee channel on the US-RSE slack before the meeting:
+
+> Note that we don't say anything about diversity in [https://us-rse.org](https://us-rse.org) or [https://us-rse.org/about/](https://us-rse.org/about/) or [https://us-rse.org/mission/](https://us-rse.org/mission/)  I think we probably should have increasing diversity of the US-RSE community (not our just members, but the larger community) as a goal.
+
+While DEI wasn't a new idea as US-RSE had talked about diversity from an early point, it was a gap in our community to not explicitly do anything about it. These early discussions led to diversity, equity, and inclusion being the topic of our [October 2020 community call](https://us-rse.org/2020-10-13-community-call-dei/), and this in turn led to the DEI slack channel and the DEI working group led by Lance Parsons. 
+The working group put together a DEI mission statement, with contributions from the entire group and leadership on the part of Alex Koufos, and an addition of DEI to the US-RSE Mission Statement.
+
+In January 2021, community members Sajid Ali took the lead to establish a DEI Speaker Series [[ref](https://docs.google.com/document/d/1ezlG5ys40LMIlv83OtFfbviHpTY5EgFM86m6vVJkTPE/edit#)], which runs quarterly.
+In May, Nicole Brewer initiated the DEI book club club [[ref](https://docs.google.com/document/d/1GMMwjdq97_AFjAuDhaJ1FtlTc3weHerH6jHaB04bN3w/edit)] that started with a reading of <span style="text-decoration:underline;">Better Allies: Everyday Actions to Create Inclusive, Engaging Workplaces” by Karen Catlin</span> which was later expanded by Miranda Mundt to include all forms of media, such as podcasts.
```

Comment:
```suggestion
In May, Nicole Brewer initiated the DEI book club club [[ref](https://docs.google.com/document/d/1GMMwjdq97_AFjAuDhaJ1FtlTc3weHerH6jHaB04bN3w/edit)] that started with a reading of <span style="text-decoration:underline;">Better Allies: Everyday Actions to Create Inclusive, Engaging Workplaces” by Karen Catlin</span>, and which was later expanded by Miranda Mundt to include all forms of media, such as podcasts.
```

Reply:
This reads weird to me, but the original sentence was off too. What about a period, and then "The book club was later expanded..." and get rid of the which?

Reply:
ok